pkgs/data/fonts/alice/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation (attrs: {
+  pname = "Alice";
+  version = "2.003";
+
+  outputs = [ "out" "woff2" ];
+
+  src = fetchzip {
+    url = with attrs; "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```

Well, I do not like to use `with`, so I suggest this below:

```suggestion
    url = let 
      inherit (attrs) pname version;
    in "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```

Or maybe:

```suggestion
    url = with attrs;
       "https://github.com/cyrealtype/${pname}/releases/download/v${version}/${pname}-v${version}.zip";
```

---

I think such a local use of `with` is fine.